packages/manager/src/features/CloudPulse/Alerts/AlertsLanding/AlertsDefinitionLanding.tsx

```diff
@@ -1,14 +1,14 @@
-import { Paper, Typography } from '@linode/ui';
 import * as React from 'react';
 import { Route, Switch } from 'react-router-dom';
 
+import { AlertListing } from '../AlertsListing/AlertListing';
 import { CreateAlertDefinition } from '../CreateAlert/CreateAlertDefinition';
 
 export const AlertDefinitionLanding = () => {
   return (
     <Switch>
       <Route
-        component={AlertDefinition}
+        component={() => <AlertListing />}
```

Comment:
I think we can do this here:

```suggestion
        component={AlertListing}
```